packages/examples/react-webpack/src/app/components/testTimepicker/TestTimepicker.tsx

```diff
@@ -0,0 +1,21 @@
+import { OdsTimepicker } from '@ovhcloud/ods-components/react';
+import { JSX } from '@ovhcloud/ods-components';
+import React, { HTMLAttributes, RefAttributes } from 'react';
+
+const TestTimepicker: React.FC<
+  Partial<
```

Comment:
Test when defining custom props and using previous `OdsTimezonePreset` type on one.